lib/samples/set_basemap/set_basemap_sample.dart

```diff
@@ -47,7 +51,7 @@ class _SetBasemapSampleState extends State<SetBasemapSample>
   @override
   void initState() {
     super.initState();
-
+    // Set the initial viewpoint and basemap for the map.
     _arcGISMap.initialViewpoint = _sanFranciscoViewpoint;
     _mapViewController.arcGISMap = _arcGISMap;
     _loadBasemapsFuture = loadBasemaps();
```

Comment:
I tried this but for some reason when you put it in onMapViewReady it never loads the basemaps. Maybe because it builds the Drawer first before it reaches the onMapViewReady call?

Reply:
You might be right. Probably `_loadBasemapsFuture` has to remain in `initState()`. So yeah, keep `initState()` around just for the `_loadBasemapsFuture` line. The other stuff can move to `onMapViewReady()`.